server/api/users/users.test.ts

```diff
@@ -72,4 +72,96 @@ describe('auth router', () => {
       });
     });
   });
+
+  describe('PATCH /:id', () => {
+    it('returns 403 if profile id is not equal to logged in user id', async () => {
+      const res = await server.exec.patch('/api/users/54321');
+      expect(res.status).toBe(403);
+    });
+
+    it('returns 404 if specified user doesn\'t exist', async () => {
+      const res = await server.exec.patch('/api/users/1');
```

Comment:
wouldn't an id of `1` possibly attempt this action on a user that exists, since user IDs are autoincrementing starting from 1? 😅